packages-web/badge/tests/e2e/specs/Badge.spec.ts

```diff
@@ -6,43 +6,15 @@ describe("badge-web", () => {
     beforeAll(() => {
         page.open(); // resets page
     });
-
     describe("type: badge", () => {
-        it("should render a badge with a static caption", () => {
-            const badge = new badgeWidget("badgeStaticSuccess");
-            expect(badge.getBadgeText()).toEqual("Static");
-        });
-
-        it("renders with brand style success", () => {
-            const badge = new badgeWidget("badgeStaticSuccess");
-            expect(badge.getColors()).toEqual(badge.defaultStyles.SuccessBackground);
-        });
-
-        it("renders with brand style info", () => {
-            const badge = new badgeWidget("badgeInfo");
-            expect(badge.getColors()).toEqual(badge.defaultStyles.InfoBackground);
-        });
-
-        it("renders with brand style default", () => {
-            const badge = new badgeWidget("badgeDefault");
-            expect(badge.getColors()).toEqual(badge.defaultStyles.DefaultBackground);
+        it("compares successfully with a screenshot baseline", () => {
+            // Change the window size
```

Comment:
I think code is pretty self explanatory, so you dont need additional comments here and line 13,15

Reply:
Removed!